docs/source/commandline/commandline-all.rst

```diff
@@ -6,6 +6,16 @@ Universal Commandline Options
 
 This section contains configuration options that are applicable to *all* MAME sub-builds (both SDL and Windows native).
 
+
+Patterns
+--------
+
+Many commands support the use of *patterns*, which are either a system or device short name (e.g. **a2600**, **zorba_kbd**) or a glob pattern that matches either (e.g. **zorba_\***)
```

Comment:
Remember the distinction between "verbs" like `-verifyroms` and `-listxml` "commands" like `srcclean`, `romcmp` and `mame`.  Also you should probably note that glob patterns should be quoted to avoid having the shell try to expand them against file names.

Reply:
Fixed on my side.